apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/model/Mcs.java

```diff
@@ -73,6 +73,8 @@ public class Mcs {
   public static final int MAX_DIAGNOSIS_CODES =
       Math.min(ICD_DGNS_CD.size(), ICD_DGNS_VRSN_CD.size());
 
+  /*public static final String DTL_NDC = CarrierClaimColumn.LINE_NDC_CD.name();*/
```

Comment:
Do we need this or should this line be deleted?

Reply:
@aschey-forpeople Not needed! This is when I attempted to map DTL_NDC to a field from CarrierClaim but turns out Synthea does not generate values for that field so the DTL_NDC doesn't get mapped. I decided to hardcode it instead.